code/game/objects/items/cigs_lighters.dm

```diff
@@ -916,6 +916,60 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	overlay_state = "slime"
 	grind_results = list(/datum/reagent/iron = 1, /datum/reagent/fuel = 5, /datum/reagent/medicine/pyroxadone = 5)
 
+/obj/item/lighter/skull
+	name = "badass zippo"
+	desc = "An absolutely badass zippo lighter. Just look at that skull!"
+	custom_price = PAYCHECK_CREW * 6
```

Comment:
This doesn't give it a sale value, you'll need to make a cargo export datum for it to be worth that much in cargo.
Check the `cargo/exports` folder for examples.